amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2446,6 +2808,25 @@ parameters:
     items:
       type: string
     collectionFormat: csv
+  # Holds
+  holdIdParam:
+    name: holdId
+    in: path
+    description: The identifier of a hold.
+    required: true
+    type: string
+  reasonParam:
+    name: reason
+    in: query
+    description: Reason for deletion.
```

Comment:
Reason for hold?

Reply:
Yes. This is an optional parameter to add a reason for hold deletion.

Reply:
[Added a separate endpoint to delete a hold with a reason.](https://github.com/Alfresco/alfresco-community-repo/pull/2514/commits/18fc9a15cd5e5da9d7df54d32bc916ffb8c62e72)